apps/src/craft/code-connection/cc-client.js

```diff
@@ -4,10 +4,27 @@ import $ from 'jquery';
  * CC client for 3rd party extension
  *
  */
+
 const baseUrl = "http://localhost:";
 export default class cc_client {
+
+    callbackRef = null;
+    keyRef = null;
+
     constructor(port) {
         this.port = port;
+        if (window.ipcRenderer !== undefined) {
+            window.ipcRenderer.on('responseFromApp', (event, arg) =>{
+                let response = JSON.parse(arg);
```

Comment:
nit; I'd love to see something more descriptive than "arg" for this argument

Reply:
Roger that! It's basically the response from the Code Connection app in JSON format. I can easily make that more explicit.